undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -387,7 +387,7 @@ undertow.application-security-domain.http-authentication-factory=The HTTP Authen
 undertow.application-security-domain.override-deployment-config=Should the authentication configuration in the deployment be overridden by the factory.
 undertow.application-security-domain.enable-jacc=Enable authorization using JACC
 undertow.application-security-domain.enable-jaspi=Enable JASPI authentication for the associated deployments.
-undertow.application-security-domain.integrated-jaspi=Should integrated JASPI be used or allow AsHoc identity creation.
+undertow.application-security-domain.integrated-jaspi=Whether integrated JASPI should be used. If this is disabled, ad hoc identity will be created instead.
```

Comment:
s/ad hoc identity/an ad hoc identity/g

Also, if this is a boolean is "If 'false'" better than "If this is disabled"?

Reply:
Thank you, Brian, for your reivew. I have updated PR based on your comment. It's ready for re-review now.